src/pudl/transform/eia923.py

```diff
@@ -1223,3 +1223,96 @@ def _core_eia923__fuel_receipts_costs(
     frc_df = frc_df.dropna(subset=["energy_source_code"])
 
     return frc_df
+
+
+@asset
+def _core_eia923__fgd_operation_maintenance(
+    raw_eia923__fgd_operation_maintenance: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transforms the _core_eia923__fgd_operation_maintenance table.
+
+    Transformations include:
+
+    * Drop values with plant and boiler id values of NA.
+    * Replace . values with NA.
+    * Convert dollars to thousands of dollars.
```

Comment:
I think this is backwards - should be thousands of dollars to dollars?

Reply:
Oh definitely, good catch